__tests__/app.test.js

```diff
@@ -142,4 +142,52 @@ describe('app', () => {
         })
 
     })
-})
\ No newline at end of file
+    describe('GET /api/articles/:article_id/comments', () => {
+        test('200: responds with status 200 for successful request', () => {
+            return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                })
+
+        
+        test('responds with an array of comment objects each with the required properties', () => {
+            return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then((response) => {
+                    const comments = Object.keys(response.body.comments[0])
+
+                    expect(comments).toEqual(['comment_id', 'body', 'article_id', 'author', 'votes', 'created_at' ])
+                })
+        })
+        test('responds with an array of comments of the expected length sorted by date in ascending order', () => {
+            return request(app)
+            .get('/api/articles/1/comments')
+            .expect(200)
+            .then((response) => {
+                const comments = response.body.comments
+                expect(comments).toHaveLength(11)
+                expect(comments).toBeSortedBy('created_at', { descending: false })
+            })
+        })
+        test('404: responds with status 404 for an article id that has no comments', () => {
```

Comment:
We can add a test for when there are no comments for a particular article. hint: think about when you read an article on bbc news - if the article has no comments, you dont want to see an error just because nobody has commented on it right? An error will prevent us from loading the page and reading the article - in which case no new article would ever receive a comment because we'd never be able to read it in the first place! So you want to say the request for the comments was successful - you just didnt get any comments back